.github/workflows/lint.yaml

```diff
@@ -26,4 +29,5 @@ jobs:
           version: v1.38
```

Comment:
I think if you bump this to 1.4X is should work

Reply:
running the 1.4x locally, there are several issues that were fixed in v0.35.x